src/Playwright.Tests/Helpers/DoubleExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using System;
```

Comment:
I assume this is here because you moved the one in `Playwright` to `internal`, which makes sense. I'd almost prefer we use `InternalsVisibleTo` and we can avoid code duplication. I can fix that separately after we merge this one though.

Reply:
We removed the `InternalsVisibleTo` flag because we were testing internal APIs that were supposed to be public.